src/m365/spe/commands/containertype/containertype-list.ts

```diff
@@ -0,0 +1,68 @@
+import { Logger } from '../../../../cli/Logger.js';
+import config from '../../../../config.js';
+import request, { CliRequestOptions } from '../../../../request.js';
+import { ClientSvcResponse, ClientSvcResponseContents, FormDigestInfo, spo } from '../../../../utils/spo.js';
+import SpoCommand from '../../../base/SpoCommand.js';
+import commands from '../../commands.js';
+import { ContainerTypeProperties } from '../../ContainerTypeProperties.js';
+
+class SpeContainertypeListCommand extends SpoCommand {
+  private allContainerTypes?: ContainerTypeProperties[];
+
+  public get name(): string {
+    return commands.CONTAINERTYPE_LIST;
+  }
+
+  public get description(): string {
+    return 'Lists all Container Types';
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['ContainerTypeId', 'ContainerTypeName', 'OwningApplicationId'];
+  }
+  public async commandAction(logger: Logger): Promise<void> {
+    try {
+      const spoAdminUrl = await spo.getSpoAdminUrl(logger, this.debug);
+
+      if (this.verbose) {
+        await logger.logToStderr(`Retrieving list of Container types...`);
+      }
+
+      this.allContainerTypes = [];
+
+      await this.getAllContainerTypes(spoAdminUrl, logger);
+
+      await logger.log(this.allContainerTypes);
+    }
+    catch (err: any) {
+      this.handleRejectedPromise(err);
+    }
+  }
+
+  private async getAllContainerTypes(spoAdminUrl: string, logger: Logger): Promise<void> {
+    const formDigest: FormDigestInfo | undefined = undefined;
+    const formDigestInfo: FormDigestInfo = await spo.ensureFormDigest(spoAdminUrl, logger, formDigest, this.debug);
```

Comment:
I think we may just past `undefined` we don't need to create a fake variable for that

```suggestion
    const formDigestInfo: FormDigestInfo = await spo.ensureFormDigest(spoAdminUrl, logger, undefined, this.debug);
```